sbking_checkstyle.xml

```diff
@@ -161,16 +161,25 @@
 
     <!-- Checks for class design                         -->
     <!-- See https://checkstyle.org/config_design.html -->
-    <module name="DesignForExtension"/>
+    <module name="DesignForExtension">
+      <property name="severity" value="warning"/>
```

Comment:
Probably info is better. I really don't care much about that specific check.. Anyway, your call